### ISSUE TITLE
Permit copying data created in one template in Sinopia into another template

### ISSUE THREAD

**Issue by jsoenyun** [LD4P/sinopia_editor#1697]
**Describe the feature you'd like**
Develop the Copy feature so that data created in one profile or template can be ported into another profile or template

**Give an example**
If dealing with an online instance of a print resource that has already been described, it would be useful to copy the data for the print instance into a different template tailored specifically for an online instance.

**Describe alternatives you've considered**
@michelleif has mentioned a workaround where the RDF could be copied and used for a new description after deleting the name of the original template, causing the system to prompt for template name. Manually re-entering the data would be another but more laborious option.

**Comment by justinlittman** [comment]
Options include:
* Asking the user when copying which RT to use.
* Allowing the user to change the RT for the currently open RT.

I personally like the second option, as it is more generally usable.

**Comment by jsoenyun** [comment]
@justinlittman agreed, sounds like much less hassle to change than to have to select from the list

**Comment by justinlittman** [comment]
@astridu Design needed. This would be a button / link on the editor tab that would open a modal to allow the user to change the resource template. (I can imagine including as another button in the Save, Copy, Preview row or perhaps next to the resource template name.)
